test/Answer.test.js

```diff
@@ -0,0 +1,117 @@
+process.env.NODE_ENV = 'test';
+const chai = require('chai');
+const expect = chai.expect;
+const sample_data = require('./test.data.json');
+
+const supertest = require('supertest');
+const app = require('../index');
+
+//positive test for post answer
+describe('Answer Test', function () {
+    describe('Positive Tests', function () {
+        it('should successfully post an answer', async function () {
+            const response = await supertest(app)
+                .post('/api/v1/answer')
+                .set({
+                    Authorization: sample_data.auth_token,
+                    Accept: 'application/json',
+                })
+                .send({
+                    answer_text: 'grace',
+                    user_id: '24',
+                    question_id: '19',
+                })
+                .expect(200);
+            const resp_data = response.body;
+            expect(resp_data).to.be.an('object');
+            expect(resp_data).to.have.property('user_id');
+            expect(resp_data.user_id).to.be.an('number');
+        });
+    });
+
+    //positive test for viewing answer
+    describe('Positive Tests', function () {
```

Comment:
Since the next test case is still part of the `Positive Test` suites then no need to use another `describe`, so remove line:30-33

Reply:
You have not remove line 30-33 as recommended above